sections/collage.liquid

```diff
@@ -2,9 +2,36 @@
 {{ 'component-card.css' | asset_url | stylesheet_tag }}
 {{ 'component-price.css' | asset_url | stylesheet_tag }}
 
+{% liquid
+  assign section_mobile_padding_top = section.settings.padding_top
+  assign section_mobile_padding_bottom = section.settings.padding_bottom
+
+  if section.settings.padding_top >= 28 or section.settings.padding_top <= 80
+    assign section_mobile_padding_top = section.settings.padding_top | minus: 10
+  endif
+
+  if section.settings.padding_bottom >= 28 or section.settings.padding_bottom <= 80
+    assign section_mobile_padding_bottom = section.settings.padding_bottom | minus: 10
+  endif
+%}
+
+{%- style -%}
+  .section-{{ section.id }}-padding {
+    padding-top: {{ section_mobile_padding_top }}px;
+    padding-bottom: {{ section_mobile_padding_bottom }}px;
+  }
+
+  @media screen and (min-width: 750px) {
+    .section-{{ section.id }}-padding {
+      padding-top: {{ section.settings.padding_top }}px;
+      padding-bottom: {{ section.settings.padding_bottom }}px;
+    }
+  }
+{%- endstyle -%}
```

Comment:
Any reason we couldn't make all this a snippet to be re-used in each section? Or are you trying to avoid the overuse of snippets? I feel like only having to make updates/fixes in one place would be cleaner and easier than a big find/replace.

Reply:
Bonus tip, by the way - you can do multi-line find/replace, so when the code is 100% identical between the different sections, you can actually get rid of whole big blocks like that.

Reply:
For now we've removed all of the math.  I will be making a follow-up PR that will explore using some fancy math to create a smoother curve.